freqtrade/tests/test_acl_pair.py

```diff
@@ -53,7 +53,7 @@ def test_refresh_whitelist(mocker):
     whitelist = ['BTC_ETH', 'BTC_TKN']
     pairslist = conf['exchange']['pair_whitelist']
     # Ensure all except those in whitelist are removed
-    assert_list_equal(whitelist, pairslist)
```

Comment:
you still need the assert statement though: `assert set(...) == set(...)`

Reply:
Thanks! I've changed it.
Should I merge all my commits into a single one, and resubmit a new PR?

Reply:
@kryofly Thanks a lot! Don't worry about that, with github it's possible to do exactly that during merge